imaginary.go

```diff
@@ -84,6 +86,7 @@ Options:
   -enable-url-source        Enable remote HTTP URL image source processing
   -enable-placeholder       Enable image response placeholder to be used in case of error [default: false]
   -enable-auth-forwarding   Forwards X-Forward-Authorization or Authorization header to the image source server. -enable-url-source flag must be defined. Tip: secure your server from public access to prevent attack vectors
+  -custom-headers			Forwards custom headers to the image source server. -enable-url-source flag must be defined.
```

Comment:
The same thing about the white-space. I think replacing the tabs with spaces cleans it up nicely.